src/services/state-persistor/utils/FeatureHash.ts

```diff
@@ -0,0 +1,1399 @@
+import ngeoFormatFeatureProperties from './FeatureProperties'
+import { rgbArrayToHex } from './utils'
+
+import olFeature from 'ol/Feature.js'
+import olMap from 'ol/Map.js'
+import { includes as arrayIncludes } from 'ol/array.js'
+import { asArray as colorAsArray } from 'ol/color.js'
+import { ReadOptions, transformGeometryWithOptions } from 'ol/format/Feature.js'
+import olGeomGeometryLayout from 'ol/geom/GeometryLayout.js'
+import olGeomLineString from 'ol/geom/LineString.js'
+import olGeomMultiLineString from 'ol/geom/MultiLineString.js'
+import olGeomMultiPoint from 'ol/geom/MultiPoint.js'
+import olGeomMultiPolygon from 'ol/geom/MultiPolygon.js'
+import olGeomPoint from 'ol/geom/Point.js'
+import olGeomPolygon from 'ol/geom/Polygon.js'
+import olStyleCircle from 'ol/style/Circle.js'
+import olStyleFill from 'ol/style/Fill.js'
```

Comment:
we don't add the trailing `.js` in the lux project, why doing so now?

Reply:
yes, removed